ambry-store/src/main/java/com.github.ambry.store/DiskIOScheduler.java

```diff
@@ -0,0 +1,59 @@
+package com.github.ambry.store;
```

Comment:
I realized I was only using `./gradlew test`, which doesn't run the license check.